packages/@webex/webex-core/src/lib/services-v2/services-v2.js

```
@@ -0,0 +1,973 @@
+              reqId,
+              token,
+              activationOptions,
+              preloginUserId,
+            }),
+          ])
+        )
+        .then(([rto, user]) => ({...rto, user}))
+        .catch((error) => {
+          const response = {
+            statusCode: error.statusCode,
+            responseText: error.body && error.body.message,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)